Formula/beancount.rb

```diff
@@ -0,0 +1,199 @@
+class Beancount < Formula
+  include Language::Python::Virtualenv
+
+  desc "Double-entry accounting tool that works on plain text files"
+  homepage "http://furius.ca/beancount/"
+  url "https://files.pythonhosted.org/packages/69/ea/ddf75e1e88aaba807bea3ac101b0e226ddc60832ecfe83ad05cc3a8fff4a/beancount-2.2.3.tar.gz"
+  sha256 "1554adfd773d12cb88fd7f4da67fcb608665a9bdedc7e44834e059d1b3a08e5d"
+  depends_on "python"
```

Comment:
Can you try with this
```suggestion
  depends_on "python@3.8"
```